app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -217,7 +217,9 @@ class GithubWebhook extends RequestHandler<Body> {
     bool needsTests = false;
 
     await for (PullRequestFile file in files) {
-      if (file.filename.endsWith('.dart')) {
+      if (file.filename.endsWith('.dart') &&
+          !file.filename.startsWith('dev/devicelab/') &&
```

Comment:
What about `dev/devicelab/bin/` and `dev/devicelab/lib/tasks/`?

There's still framework code where we would tests written

Reply:
How about I just make this `dev/devicelab/bin/tasks` and also remove `dev/bots/`?  Those are the actual integration tests.

Reply:
SGTM. Is the goal to one day have tests for `dev/bots` code?

Reply:
Not a goal of mine 😉  You're saying you're cool with me excluding `dev/bots/`?  I feel like the integration themselves exercise that code.

Reply:
Oh I see what you mean. Yes i'm okay with removing `dev/bots` from asking for tests